dev/bots/test/analyze_sample_code_test.dart

```diff
@@ -20,23 +20,23 @@ void main() {
       'In sample starting at known_broken_documentation.dart:117:bool? _visible = true;',
       '>>> info: Use late for private members with non-nullable type (use_late_for_private_fields_and_variables)',
       'In sample starting at known_broken_documentation.dart:117:      child: Text(title),',
-      '>>> error: The final variable \'title\' can\'t be read because it is potentially unassigned at this point (read_potentially_unassigned_final)',
+      ">>> error: The final variable 'title' can't be read because it is potentially unassigned at this point (read_potentially_unassigned_final)",
       'known_broken_documentation.dart:30:9: new Opacity(',
       '>>> info: Unnecessary new keyword (unnecessary_new)',
       'known_broken_documentation.dart:62:9: new Opacity(',
       '>>> info: Unnecessary new keyword (unnecessary_new)',
-      'known_broken_documentation.dart:95:9: const text0 = Text(\'Poor wandering ones!\');',
+      "known_broken_documentation.dart:95:9: const text0 = Text('Poor wandering ones!');",
       '>>> info: Specify type annotations (always_specify_types)',
-      'known_broken_documentation.dart:103:9: const text1 = _Text(\'Poor wandering ones!\');',
+      "known_broken_documentation.dart:103:9: const text1 = _Text('Poor wandering ones!');",
       '>>> info: Specify type annotations (always_specify_types)',
-      'known_broken_documentation.dart:111:9: final String? bar = \'Hello\';',
+      "known_broken_documentation.dart:111:9: final String? bar = 'Hello';",
       '>>> info: Prefer const over final for declarations (prefer_const_declarations)',
-      'known_broken_documentation.dart:111:23: final String? bar = \'Hello\';',
+      "known_broken_documentation.dart:111:23: final String? bar = 'Hello';",
       '>>> info: Use a non-nullable type for a final variable initialized with a non-nullable value (unnecessary_nullable_for_final_variable_declarations)',
       'known_broken_documentation.dart:112:9: final int foo = null;',
       '>>> info: Prefer const over final for declarations (prefer_const_declarations)',
       'known_broken_documentation.dart:112:25: final int foo = null;',
-      '>>> error: A value of type \'Null\' can\'t be assigned to a variable of type \'int\' (invalid_assignment)',
+      ">>> error: A value of type 'Null' can't be assigned to a variable of type 'int' (invalid_assignment)",
       '',
```

Comment:
Having this code flip between double quotes and single quotes like this is really confusing.
Are we sure we want this lint enabled?

Reply:
I actually find it much easier to read without the escapes, and I hardly notice the difference in the double/single quotes.

Reply:
I second Greg. I find it makes the actual string (the thing I care about) easier to read.

Reply:
@goderbauer what you think about it?

Am i continue working on it??